src/pages/index.js

```diff
@@ -70,9 +100,24 @@ const previewModalElement = document.querySelector("#preview-modal");
 const previewModalImage = previewModalElement.querySelector(".modal__image");
 const previewModalCaption =
   previewModalElement.querySelector(".modal__caption");
-const previewModalCloseButton = previewModalElement.querySelector(
-  ".modal__button-close"
-);
+//const previewModalCloseButton = previewModalElement.querySelector(
+//  ".modal__button-close"
+//);
+
+// Avatar form elements
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarButton = document.querySelector(".profile__avatar-button");
+const avatarForm = avatarModal.querySelector(".modal__form");
```

Comment:
**COULD BE IMPROVED**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms